config/sys.config

```diff
@@ -76,6 +76,20 @@
                     {127,0,0,1}, 31341} },
    {default_routers, ["/p2p/112qB3YaH5bZkCnKA5uRH7tBtGNv2Y5B4smv1jsmvGUzgKT71QpE", "/p2p/1124CJ9yJaHq4D6ugyPCDnSBzQik61C1BqD9VMh1vsUmjwt16HNB"]},
    {mark_mods, [miner_hbbft_handler]},
-   {stabilization_period_start, 187000}
+   {stabilization_period_start, 187000},
+   {reg_domains_file, "countries_reg_domains.csv"},
+   {frequency_data, #{'US915' => [903.9, 904.1, 904.3, 904.5, 904.7, 904.9, 905.1, 905.3],
+                      'EU868' => [867.1, 867.3, 867.5, 867.7, 867.9, 868.1, 868.3, 868.5, 868.8],
```

Comment:
Aren't there 9 channels here?

Reply:
868.8 prob should not be there, its listed as  FSK on things network